src/views/states/list/components/InterfacesTypeSection.tsx

```diff
@@ -45,9 +46,14 @@ const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = memo(
 
           return (
             <Tr key={iface.name} isExpanded={isExpanded}>
-              <Td className="pf-m-width-20">
+              <Td className="pf-m-width-30">
                 <Button variant={ButtonVariant.link} onClick={() => setSelectedInterface(iface)}>
                   {iface.name}
+                  {iface.state.toLowerCase() === 'up' ? (
+                    <LongArrowAltUpIcon color="black" className="pf-u-mr-sm" />
+                  ) : (
+                    <LongArrowAltDownIcon color="black" className="pf-u-mr-sm" />
+                  )}
```

Comment:
consider doing before the return something like: 
`const Icon = iface.state.toLowerCase() === 'up' ? LongArrowAltUpIcon : LongArrowAltDownIcon;`
and here you could do:
<Icon color="black" className="pf-u-mr-sm" />